draft-ietf-mls-protocol.md

```diff
@@ -1933,6 +2121,19 @@ Update for the leaf if there are no Removes. If there are multiple Add proposals
 for the same client, the committer again chooses one to include and considers
 the rest invalid.
 
+If the committer recieved multiple PSK proposals, they MUST include all of them
+in an ordering of their choice.
```

Comment:
I would strike this.  It's up to the committer to decide what they commit.  For example, if someone proposes adding a nonsense PSK, that shouldn't get committed.